sunpy/map/sources/psp.py

```diff
@@ -20,17 +23,43 @@ class WISPRMap(GenericMap):
     * `WISPR Instrument Page <https://wispr.nrl.navy.mil//>`__
     * `Instrument Paper <https://doi.org/10.1007/s11214-014-0114-y>`__
     """
+
+    def __init__(self, data, header, **kwargs):
+        super().__init__(data, header, **kwargs)
+
+        self.plot_settings['norm'] = ImageNormalize(
+            stretch=AsinhStretch(a=0.001), clip=True,
```

Comment:
Why are you choosing to set `clip=True`?  That is more aggressive than merely setting `vmin`.

Reply:
Good catch, I think this was just left from the initial, more opinionated plot settings.